frontend/packages/ux-editor/src/utils/formLayoutUtils.test.tsx

```diff
@@ -628,14 +628,26 @@ describe('formLayoutUtils', () => {
       const layouts: Record<string, IInternalLayout> = {
         page1: {
           order: { section1: ['component1'] },
-          components: {},
+          components: {
+            component1: {
+              id: 'component1',
+              type: ComponentType.Paragraph,
+              itemType: 'COMPONENT',
+            },
+          },
           containers: {},
           customRootProperties: {},
           customDataProperties: {},
         },
         page2: {
           order: { section1: ['component2'] },
-          components: {},
+          components: {
+            component2: {
+              id: 'component2',
+              type: ComponentType.Paragraph,
+              itemType: 'COMPONENT',
+            },
+          },
```

Comment:
Are these changes needed?

Reply:
No actualy, after we fixed the other tests, it works witout needing to change it, i reverted to old 👍